test/higher_order_ops/test_invoke_quant.py

```diff
@@ -0,0 +1,88 @@
+# Owner(s): ["module: higher order operators"]
+# flake8: noqa: B950
+
+import contextlib
+import logging
+
+import torch
+import torch._dynamo
+import torch._functorch
+import torch._inductor
+import torch._inductor.decomposition
+from torch._higher_order_ops import InvokeQuant
+from torch.testing import FileCheck
+from torch.testing._internal.common_utils import run_tests, skipIfTorchDynamo, TestCase
+
+
+invoke_quant_tracer = InvokeQuant()
+
+
+@skipIfTorchDynamo("Not a torch._dynamo test")
+class TestInvokeQuant(TestCase):
+    backend = ""
+
+    def test_simple(self):
+        def gn(x, y):
+            return (torch.mul(x, y) + y,)
+
+        def fn(x, y):
+            return invoke_quant_tracer(gn, (x, y), scheme="nf4")[0]
+
+        x = torch.randn(8, requires_grad=False)
+        y = torch.randn(8, requires_grad=False)
+        ref = gn(x, y)[0]
+
+        x_clone = x.clone().detach().requires_grad_(False)
+        y_clone = y.clone().detach().requires_grad_(False)
+        res = torch.compile(fn, backend=self.backend)(x_clone, y_clone)
+        self.assertEqual(ref, res)
```

Comment:
can you do a self.assertExpectedInline on the output graph from Dynamo?

Reply:
Ditto on an output graph from AOTDispatcher. There's EagerWithGraphs and AOTEagerWithGraphs backend to help with that